### DISCUSSION TITLE
Rename Performances to Performance in docs

### DISCUSSION THREAD

**Discussion by andyexeter** [lhapaipai/symfony-vite-dev#48] [General]
Hi @lhapaipai,

I think we should rename https://symfony-vite.pentatrion.com/guide/performances.html from Performances to Performance - it makes more sense to use the singular form in English.

If we make this change, is it possible to redirect https://symfony-vite.pentatrion.com/guide/performances.html to https://symfony-vite.pentatrion.com/guide/performance.html so the old URL still works?

**Comment by lhapaipai** [comment]
hello @andyexeter, indeed I think I am influenced by the use of this word (which is the same) in French,

**For the English version**, you can of course rename the file name and title to whatever you think is most appropriate.

**For the French version**,
can you rename the filename so that it is the same as the English version (to remain consistent with the other file names) but on the other hand leave the title `# Performances` because it does not really have a singular meaning in French.

the documentation is hosted by netlify so you can update the `netlify.toml` file at the root of the repository as you wish : https://docs.netlify.com/routing/redirects/#syntax-for-the-netlify-configuration-file

Have a good day

**Reply by lhapaipai** [reply]
so I made the modification, if it doesn't suit to you, you can modify :-)
good evening
